html/lists.html

```diff
@@ -1,43 +1,45 @@
 <html>
+
 <head>
-<style>
-ul {
-  list-style-type:square;
-  margin: 2px;
-  padding: 3px;
-  overflow: hidden;
-  background-color: #01646b9d;
-}
-
-li {
-  float:left;
-}
-
-li a {
-  display: block;
-  color: white;
-  text-align: center;
-  padding: 16px;
-  text-decoration: lightgray;
-}
-
-li a:hover {
-  background-color: #111111;
-}
-</style>
+  <style>
+    ul {
+      list-style-type: square;
+      margin: 2px;
+      padding: 3px;
+      overflow: hidden;
+      background-color: #01646b9d;
+    }
+
+    li {
+      float: left;
+    }
+
+    li a {
+      display: block;
+      color: white;
+      text-align: center;
+      padding: 16px;
+      text-decoration: lightgray;
+    }
+
+    li a:hover {
+      background-color: #111111;
+    }
+  </style>
 </head>
+
 <body>
 
-<h2>Navigation Menu</h2>
-<p></p>
+  <h2>Navigation Menu</h2>
+  <p></p>
 
-<ul>
-  <li><a href="#name">Name</a></li>
-  <li><a href="#age">Age</a></li>
-  <li><a href="#height">Height</a></li>
-  <li><a href="#contact">Contact</a></li>
-  <li><a href="#Points">Points</a></li>
-</ul>
+  <ul>
+    <li><a href="#name">Name</a></li>
+    <li><a href="#age">Age</a></li>
+    <li><a href="#height">Height</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#Points">Points</a></li>
+  </ul>
 </body>
-</html>
 
+</html>
```

Comment:
If I pull your code an try this list menu, it will work if I click on the navigation options ? (Name, Age .. etc)

Reply:
you can just move from one to another,but it doesn't have a link, you cant add nothing to it, it's just a list

Reply:
So, is not just a list in the w3school context, actually #name or #age can reference to a different section or page section.

Reply:
exactly